flexget/plugins/sites/rmz.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+
+import logging
+import re
+
+from flexget import plugin
+from flexget.event import event
+from flexget.plugins.internal.urlrewriting import UrlRewritingError
+from flexget.utils.soup import get_soup
+from flexget.entry import Entry
+from flexget.utils.search import normalize_unicode
+
+log = logging.getLogger('rmz')
+
+class UrlRewriteRmz(object):
+    """
+    rmz.cr (rapidmoviez.com) urlrewriter
+    Version 0.1
+    
+    Configuration
+
+    rmz:
+      url_re:
+        - domain\.com
+        - domain2\.org
+
+    Only add links that match any of the regular expressions listed under url_re.
+
+    If more than one valid link is found, the url of the entry is rewritten to
+    the first link found. The complete list of valid links is placed in the
+    'urls' field of the entry.
+    
+    Therefore, it is recommended, that you configure your output to use the
+    'urls' field instead of the 'url' field.
+    
+    For example, to use jdownloader 2 as output, you would use the exec plugin:
+    exec:
+      - echo "text={{urls}}" >> "/path/to/jd2/folderwatch/{{title}}.crawljob"
+    """
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'links_re': {'type': 'array', 'items': {'format': 'regexp'} }
```

Comment:
1. there's a mismatch between this value and the one in the doc (`url_re`).
2. I'm not sure why is this needed. Don't you know the TLD and format you need for the rewrite? why does the user need to input this? URL rewrite plugin do not need to rely on config.